packages/docs/developer-resources/integrations/checklist.md

```diff
@@ -0,0 +1,64 @@
+
+Addresses are similar to Ethereum addresses. When displaying and asking for user-inputted addresses, consider using and validating address checksums following the [EIP55 standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) to detect typos.
+
+### QR Codes
+
+Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals/pull/40) that aims to standardize how applications can ask wallets for transactions to avoid the user having to manually copy/paste addresses and other transaction metadata.
+
+
+## Custodian/Exchange
+
+Please read more under [Custody](./custody.md), but here is a shortened version:
+
+### Detect Transfers
+cUSD is a token that can be accessed via the ERC20 interface, however the native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to `ETH` on Ethereum.
+
+### Proof of Stake
+Users may want to participate in Celo's Proof of Stake system to help secure the network and earn rewards.
+
+### Authorized Signers
+Celo's core smart contracts use Celo's `Accounts` abstraction to allow balance-moving keys to be held in cold storage, while other keys can be authorized to vote and held in warm storage or even with the user.
+
+### Release Gold
+There is an audited `ReleaseGold` smart contract which allows for the release of `CELO` over a set schedule through which `CELO` might be distributed to a user.
+
+## Wallets
+As wallets, we are loosely defining any application that is meant as key store to allow users to interact and transfer value on the Celo platform.
+
+### Key Derivation
+Celo wallets should follow the [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) for deriving private keys from [BIP39 mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). Celo's key derivation path is at `m/44'/52752'/0'/0`. The first key typically is the `account key` that wallets should register themselves with and accept balance transfers on. The second key can be derived to be an account's `dataEncryptionKey` to allow other users on Celo to encrypt information to.
+
+### Identity Protocol
+Celo has a [lightweight identity protocol](/celo-codebase/protocol/identity) that allows users to address each other via their phone number instead of addressses that Celo wallets should implement. Since user privacy is important, Celo wallets should leverage the built-in [Phone Number Privacy protocol](/celo-codebase/protocol/identity/phone-number-privacy) to protect against large-scale harvesting of user phone numbers.
+
+### Wallet Address
+When transferring assets to an account, wallets should check the receiving account's `walletAddress` at which they want to receive funds at. Use cases might be smart contract accounts that want different recovery characteristics, but receive funds at a different address. Also, `walletAddresses` of `0x0` should indicate that the account requires a different mechanism to acquire the `walletAddress`.
```

Comment:
explain `walletAddress`

Reply:
Is that not explained in the first sentence? How would you change it?